deployments/k8s/deployment.yaml

```diff
@@ -266,7 +266,122 @@ subjects:
+  conf.yaml: |
+    application:
+      name: discovery-engine
+      network:
+        operation-mode: 1                         # 1: cronjob | 2: one-time-job
+        cron-job-time-interval: "0h0m10s"         # format: XhYmZs
+        operation-trigger: 5
+        network-log-from: "kubearmor"             # db|hubble|feed-consumer|kubearmor
+        network-log-file: "./flow.json"           # file path
+        network-policy-to: "db"                   # db, file
+        network-policy-dir: "./"
+        namespace-filter:
+        - "!kube-system"
+      system:
+        operation-mode: 1                         # 1: cronjob | 2: one-time-job
+        cron-job-time-interval: "0h0m10s"         # format: XhYmZs
+        operation-trigger: 5
+        system-log-from: "kubearmor"              # db|kubearmor|feed-consumer
+        system-log-file: "./log.json"             # file path
+        system-policy-to: "db"                    # db, file
+        system-policy-dir: "./"
+        deprecate-old-mode: true
+        namespace-filter:
+        - "!kube-system"
+        fromsource-filter:
+        - "knoxAutoPolicy"
+        
+      admission-controller:
+        generic-policy-list:
+        - "restrict-deprecated-registry"
+        - "prevent-cr8escape"
+        - "check-kernel-version"
+        - "restrict-ingress-defaultbackend"
+        - "restrict-nginx-ingress-annotations"
+        - "restrict-ingress-paths"
+        - "prevent-naked-pods"
+        - "restrict-wildcard-verbs"
+        - "restrict-wildcard-resources"
+        - "require-requests-limits"
+        - "require-pod-probes"
+        - "drop-cap-net-raw"
+
+      cluster:
+        cluster-info-from: "k8sclient"            # k8sclient|accuknox
+
+    observability: 
+      enable: true
+      cron-job-time-interval: "0h0m10s"         # format: XhYmZs
+      dbname: ./accuknox-obs.db
+      system-observability: true
+      network-observability: false
+      write-logs-to-db: false
+      summary-jobs:
+        publisher: true
+        write-summary-to-db: true
```

Comment:
```suggestion
        write-summary-to-db: false
```

Reply:
@seswarrajan This was changed because of a suggestion from @Ankurk99 https://github.com/accuknox/discovery-engine/pull/764#discussion_r1252859192. Should we change back?

Reply:
@Ankurk99 We made this change to false purposefully to reduce load in DE side. If needed, we can keep this ON later.

Reply:
@seswarrajan I understand that on SaaS we won't need it. But on the cli side, we would need to enable it for the `karmor summary` to work. 
As we are using `helm charts` for installing DE on the SaaS side, so changing it in the deployment manifest here won't have any effect on the SaaS.